include/mercury/kes_dashboard_live_tiles_cli/ITile.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * \file ITile.h
+ *
+ * \copyright Copyright (C) 2019 Kano Computing Ltd.
+ *            License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+ *
+ * \brief     TODO
+ */
+
+
+#ifndef __KES_DASHBOARD_LIVE_TILES_CLI_I_TILE_H__
+#define __KES_DASHBOARD_LIVE_TILES_CLI_I_TILE_H__
+
+
+#include <memory>
+#include <string>
+
+#include <parson.h>
+
+#include "mercury/kes_dashboard_live_tiles_cli/IDownloadable.h"
+#include "mercury/kes_dashboard_live_tiles_cli/ISerialisable.h"
+
+
+class ITile : public ISerialisable, IDownloadable {
+
+    // ISerialisable Methods.
```

Comment:
Do you need to define this if you are inheriting this from `ISerialisable` - the class is pure abstract anyway so you aren't going to be instantiating it. I'd be surprised if the compiler enforces this and it just increases the number of places that you have to make a change if you do want to change something.